CHANGELOG.md

```diff
@@ -22,17 +22,21 @@ Given the `MAJOR.MINOR.PATCH` pattern, here is how we decide to increment:
 - Scroll to top functionality on footer
 - Added `/modules/util/web-storage-proxy.js` utility module.
 - Added `/modules/util/js-loader.js` utility module.
+- Adds ESLint check for `@todo` jsdoc syntax.
```

Comment:
Realizing now I was thinking this through incorrectly, since FJ won't be merged into refresh before the next beta release. We will have a gnarly conflict once the changelog is updated on refresh, but I think that's just something we'll have to deal with.